packages/x-adapter-platform/src/types/response.types.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  Banner,
+  NextQuery,
+  Promoted,
+  Redirection,
+  RelatedTag,
+  Result,
+  Suggestion,
+  TaggingInfo
+} from '@empathyco/x-types';
+
+export interface PlatformSearchResponse {
+  banner: Contentable<PlatformBanner>;
+  catalog: PlatformCatalog;
+  direct: Contentable<PlatformRedirection>;
+  promoted: Contentable<PlatformPromoted>;
+}
+export interface Contentable<T> {
+  content: T[];
+}
+
+export interface PlatformCatalog extends Contentable<PlatformResult> {
+  facets: PlatformFacet[];
+  numFound: number;
+  spellchecked: string;
+  tagging: {
+    query: string;
+  };
+}
+
+export interface SearchResponse {
+  results: Result[];
+  facets: any[];
+  totalResults: number;
+  spellcheck: string;
+  banners: Banner[];
+  promoted: Promoted[];
+  redirections: Redirection[];
+  queryTagging: TaggingInfo;
+}
+
+export interface PlatformResult {
+  name: string;
+  averageRating?: number;
+  id: string;
+  images?: string[];
+  image: string;
+  price: number;
+  sku?: string;
+  url: string;
+  tagging: PlatformTagging;
```

Comment:
I suppose that we are not using the other values like year, color, popularity...etc